webapp/client/cypress/integration/login.spec.js

```diff
@@ -49,7 +50,8 @@ describe("Login", () => {
   context("when logged in", () => {
     beforeEach(() => {
       cy.login();
-      cy.visit("/lotse/step/start");
+      cy.visit("/");
```

Comment:
Why did you change the route here?

Reply:
Because that page doesn't have the extended footer anymore and therefore we can't test if the links are disabled or not. But going to the start page after logging in should keep the same scope of "I am logged in and I can't cancel my unlock code", wdyt?

Reply:
Yes, you are right. I did not see the tests and we can test that functionality also on the front-page 👍